### DISCUSSION TITLE
dont get 2.4 Beta to work

### DISCUSSION THREAD

**Discussion by root9191** [Leantime/leantime#1922] [General]
Hi,
i tried out several Betas now on my Webspace, but mostly i don't get it to work. After updating the Files i couldn't get to the Webpage.
Do i need to do something special, like run an Update Script or modify some Files?
Or should i just wait for the RC?

**Comment by gloriafolaron** [comment]
Hi! Did you ever get this working?

In the future, support requests can be quicker through our Discord chat as well. https://discord.gg/sugXFb79Hn
